meetup.rb

```diff
@@ -0,0 +1,72 @@
+require 'httparty'
+require 'fileutils'
+require 'colorize'
+require 'dotenv'
+
+# .env
+Dotenv.load
+meetup_api_key = ENV["MEETUP_API_KEY"]
+
+meetup_group_url = "ny-tech"
+# meetup_group_url = "RubyFTW-Fort-Worth-Ruby-User-Group"
+base_uri = "https://api.meetup.com/2/events?key=#{meetup_api_key}&group_urlname=#{meetup_group_url}&sign=true"
+response = HTTParty.get(base_uri)
+body = JSON.parse(response.body)
+
+mu = body["results"][0]
```

Comment:
Perhaps we should iterate over the results? Maybe someday we will actually plan multiple events in the future :)